packages/outputs/src/components/media/html.js

```diff
@@ -2,7 +2,14 @@
 import * as React from "react";
 
 type Props = {
+  /**
+   * The HTML string that will be rendered.
+   */
   data: string,
+  /**
+   * The media type associated with the HTML
+   * string. This defaults to text/html.
+   */
   mediaType: string
```

Comment:
In the future I think it would be nice to make these mediaType props an enum of different mimetypes.

Reply:
For individual components they should likely just take the literal types they accept. For this one, just `text/html`.

```suggestion
  mediaType: 'text/html'
```

Reply:
Nice, I just tried the whole "Apply Suggestion" inline thing. Pretty cool.